tests/msbuild-times-reference/MSBuildDeviceIntegration.csv

```diff
@@ -15,3 +15,4 @@ Build_XAML_Change,7400
 Build_XAML_Change_RefAssembly,4000
 Install_CSharp_Change,4500
 Install_XAML_Change,5000
+Install_CSharp_FromClean,20000
```

Comment:
So on CI we are averaging about 15-16 seconds for a first deploy (its something like 4 locally). 
I'm not 100% sure what the best value to put in for this is.

Reply:
Do we want to merge this one first?
* https://github.com/dotnet/android/pull/9231
I removed the 500ms it was adding to all the numbers, I think this was leftover from when we had Xamarin.Android and .NET 6+ tests.

Reply:
rebased